django_mongodb/features.py

```diff
@@ -673,6 +664,17 @@ def django_test_expected_failures(self):
             "backends.base.test_base.DatabaseWrapperLoggingTests",
             "migrations.test_operations.OperationTests.test_run_python_atomic",
         },
+        "QuerySet extras are not supported on MongoDB": {
+            # QuerySet.extra(select=...) raises NotSupportedError instead of:
+            # 'RawSQL' object has no attribute 'as_mql'.
+            "aggregation_regress.tests.AggregationTests.test_annotate_with_extra",
```

Comment:
You can put these in the existing "QuerySet.extra() is not supported." category.